airflow-core/src/airflow/ui/src/i18n/locales/zh-TW/dags.json

```
@@ -0,0 +1,147 @@
+{
+  "assetSchedule": "{{count}} / {{total}} 個資源事件已更新",
+  "dagActions": {
+    "delete": {
+      "button": "刪除 Dag",
+      "warning": "這將會刪除所有與此 DAG 相關的系統資料，包括 DAG 執行與任務。"
+    },
+    "trigger": {
+      "button": "觸發",
+      "triggerDag": "觸發 Dag"
+    }
+  },
+  "filters": {
+    "paused": {
+      "active": "啟用中",
+      "all": "全部",
+      "paused": "暫停"
+    }
+  },
+  "list": {
+    "advancedSearch": "進階搜尋",
+    "clearSearch": "清除搜尋",
+    "columns": {
+      "dagId": "Dag ID",
+      "lastDagRun": "上次 Dag 執行",
+      "nextDagRun": "下次 Dag 執行",
+      "schedule": "排程",
+      "tags": "標籤"
+    },
+    "ownerLink": "Owner link for {{owner}}",
+    "searchPlaceholder": "搜尋 Dags"
+  },
+  "runAndTaskActions": {
+    "clear": {
+      "button": "清除 {{type}}",
+      "buttonTooltip": "按下 shift+c 清除",
+      "dialog": {
+        "affectedTasks": {
+          "columns": {
+            "mapIndex": "Map Index",
```

```suggestion
            "mapIndex": "對映索引",
```

not sure whether it's best this way 🤔 

https://github.com/search?q=repo%3Apython%2Fpython-docs-zh-tw%20map&type=code